aima-core/src/main/java/aima/core/nlp/ranking/Page.java

```diff
@@ -0,0 +1,33 @@
+package aima.core.nlp.ranking;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Page {
+	
+	public double authority;
+	public double hub;
```

Comment:
It may make sense to add getters and setters for these 2 values instead of making them public. But as is works for a basic implementation.